poc/common.sage

```diff
@@ -75,38 +75,17 @@ def vec_add(left, right):
 def vec_neg(vec):
     return list(map(lambda x: -x, vec))
 
```

Comment:
Reviewer note: I plan to propagate these changes to the draft before merging this PR. For now, the PR is marked as draft until we've decided whether to reverse the IDPF index bit order (see below).